docs/freqai-data-handling.md

```diff
@@ -0,0 +1,115 @@
+|| `*_trained_dates.df.pkl` - The dates associated with the `trained_df.pkl`, which is useful for post-processing. |
+
+The example file structure would look like this:
+
+```
+├── models
+│   └── unique-id
+│       ├── config_freqai.example.json
+│       ├── historic_predictions.backup.pkl
+│       ├── historic_predictions.pkl
+│       ├── pair_dictionary.json
+│       ├── sub-train-1INCH_1662821319
+│       │   ├── cb_1inch_1662821319_metadata.json
+│       │   ├── cb_1inch_1662821319_model.joblib
+│       │   ├── cb_1inch_1662821319_pca_object.pkl
+│       │   ├── cb_1inch_1662821319_svm_model.joblib
+│       │   ├── cb_1inch_1662821319_trained_dates_df.pkl
+│       │   └── cb_1inch_1662821319_trained_df.pkl
+│       ├── sub-train-1INCH_1662821371
+│       │   ├── cb_1inch_1662821371_metadata.json
+│       │   ├── cb_1inch_1662821371_model.joblib
+│       │   ├── cb_1inch_1662821371_pca_object.pkl
+│       │   ├── cb_1inch_1662821371_svm_model.joblib
+│       │   ├── cb_1inch_1662821371_trained_dates_df.pkl
+│       │   └── cb_1inch_1662821371_trained_df.pkl
+│       ├── sub-train-ADA_1662821344
+│       │   ├── cb_ada_1662821344_metadata.json
+│       │   ├── cb_ada_1662821344_model.joblib
+│       │   ├── cb_ada_1662821344_pca_object.pkl
+│       │   ├── cb_ada_1662821344_svm_model.joblib
+│       │   ├── cb_ada_1662821344_trained_dates_df.pkl
+│       │   └── cb_ada_1662821344_trained_df.pkl
+│       └── sub-train-ADA_1662821399
+│           ├── cb_ada_1662821399_metadata.json
+│           ├── cb_ada_1662821399_model.joblib
+│           ├── cb_ada_1662821399_pca_object.pkl
+│           ├── cb_ada_1662821399_svm_model.joblib
+│           ├── cb_ada_1662821399_trained_dates_df.pkl
+│           └── cb_ada_1662821399_trained_df.pkl
+```
+
+## Live deployments
+
+### Automatic data download
+
+FreqAI automatically downloads the proper amount of data needed to ensure training of a model through the user defined `train_period_days` and `startup_candle_count` (see the [parameter table](freqai-parameter-table.md) for detailed descriptions of these parameters). 
+
+### Saving prediction data
+
+All predictions made during the lifetime of a specific `identifier` model are stored in `historical_predictions.pkl` to allow for reloading after a crash or changes made to the config.
+
+### Purging old model data
+
+FreqAI stores new model files after each successful training. These files become obsolete as new models are generated to adapt to new market conditions. The user who is planning to leave FreqAI running for extended periods of time with high frequency retraining should enable `purge_old_models` in their config:
+
+```json
+    "freqai": {
+        "purge_old_models": true,
+    }
+```
+
+This will automatically purge all models older than the two most recently trained ones to save disk space.
+
+### Returning additional info from training
```

Comment:
this segment should probably go to "feature engineering".
The way this page starts tells a new user "technical details, not interesting for me" - while this part (and the part above) would be interesting.

Reply:
Removed the 'Data handling' page all together and moved things as per comment above.